### DISCUSSION TITLE
Research mini-game similar to Atomix

### DISCUSSION THREAD

**Discussion by rene-descartes2021** [space-wizards/space-station-14#4621] [Ideaguys]
Play an old incarnation [in your browser](https://archive.org/details/msdos_Atomix_1990).
Or play a clone on your Debian based system:
```
$ apt-get install atomix
$ atomix
```
See how far you get :-)
Trivia: https://en.wikipedia.org/wiki/Atomix_(video_game)

This is a science themed game. My thinking is to frame the game in terms of calibrating a chemical synthesizer. With bonus rounds being correctly identifying chemicals by visualizing their structure (multiple choice). The points gained from the game would go towards research. More points for completing the game faster, and completing more rounds. I won't be doing the balancing, that's up to you guys.

IMO there are many initial states (levels) to put into a [TabletopSetup System](https://github.com/space-wizards/space-station-14/pull/4610/files#diff-8ea27466035ec540d988cadaad6ddd5316624b05090cda18cb9dd01479a00d80). IMO should be put into an initial-board-state component. You can browse the initial states for the GNOME clone (GPL3 licence) [here](https://gitlab.gnome.org/GNOME/atomix/-/blob/master/data/level/sequence). **EDIT:** [here](https://gitlab.gnome.org/GNOME/atomix/-/blob/master/data/level/water.atomix.xml.in) is an example of a level, they encoded in XML. But, I figure in YAML, not as MapCoordinates but grid indices which would translate into MapCoordinates by the System.

I expected to do crude custom art in doing the PR or find something with suitable CC/MIT license, unless someone else better at art would like to do that.

I'll probably use arrows for atom movement similar to that GNOME clone. I had not yet thought out what additions would need to be made to TabletopDraggableComponent or what.

This would be for more of a depth/breadth of research activities one could do, it isn't the only thing.

**Comment by PaulRitter** [comment]
Seems like a fun game. I don't know if we should make it a *core* part of the science loop, but having this as one of many things to do research with might be very fun.
